apps/etl/migrations/0002_pystacloaddata_item_datetime_pystacloaddata_item_id_and_more.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 5.1.5 on 2025-04-15 10:00
+
+import django.utils.timezone
+from django.db import migrations, models
+
+def populate_fields(apps, schema_editor):
+    Pystacmodel = apps.get_model('etl','PyStacLoadData')
+    Pystacobjects = Pystacmodel.objects.all()
+    for component in Pystacobjects:
+        data = component.item or {}
+        item_id = data.get('id')
+        item_datetime = data.get('properties',{}).get('datetime')
+        if item_id:
+            component.item_id = item_id
+
+        if item_datetime:
+            component.item_datetime = item_datetime
+
+        if item_id and item_datetime:
+            component.save()
```

Comment:
```suggestion
            component.save(update_fields=["item_id, item_datetime"])
```

Reply:
As we are updating just two field lets use partial update as suggested. calling component.save() saves all the fields of the component.